model/host/host.go

```diff
@@ -774,6 +777,22 @@ func FindAllRunningParents() ([]Host, error) {
 	return hosts, nil
 }
 
+// FindAllRunningParentsOrdered finds all running hosts with child containers,
+// sorted in order of soonest  to latest LastContainerFinishTime
+func FindAllRunningParentsOrdered() ([]Host, error) {
+	query := db.Query(bson.M{
+		StatusKey:        evergreen.HostRunning,
+		HasContainersKey: true,
+	})
+	querySorted := query.Sort([]string{LastContainerFinishTimeKey})
```

Comment:
I think these functions were written deliberately so that you could chain them:
```
	query := db.Query(bson.M{
		StatusKey:        evergreen.HostRunning,
		HasContainersKey: true,
	}).Sort([]string{LastContainerFinishTimeKey})
```